metricbeat/module/vsphere/datastore/datastore_test.go

```diff
@@ -85,9 +85,9 @@ func TestData(t *testing.T) {
 	ts := model.Service.NewServer()
 	defer ts.Close()
 
-	f := mbtest.NewReportingMetricSetV2Error(t, getConfig(ts))
+	f := mbtest.NewReportingMetricSetV2WithContext(t, getConfig(ts))
 
-	if err := mbtest.WriteEventsReporterV2Error(f, t, ""); err != nil {
+	if err := mbtest.WriteEventsReporterV2WithContext(f, t, ""); err != nil {
```

Comment:
`WriteEventsReporterV2WithContext` seems to be undefined for 7.2, maybe we should keep `WriteEventsReporterV2Error` here?

Reply:
:man_facepalming: Yeap, good catch :grimacing: